### ISSUE TITLE
EMR on EKS Anywhere

### ISSUE THREAD

**Issue by thecloudgarage** [aws/eks-anywhere#4723]
Given that IRSA is supported on EKS Anywhere., can the article EMR on EKS https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/emr-eks.html be extended to EMR on EKS Anywhere. 

If yes, is there a guidance that can be provided?

cheers

Ambar Hassani

**Comment by TerryHowe** [comment]
Have you brought this up with your technical account manager if you have one?

**Comment by thecloudgarage** [comment]
No. I don't have one. At this stage, all I am looking is to know whether it's technically feasible or not to run EMR on EKS Anywhere. If NO, then I will not pursue. If YES, we will figure out a way to make it work.

cheers,

Ambar Hassani
